integration/ct_config.sh

```diff
@@ -1,18 +1,36 @@
 #!/bin/bash
-# Builds configuration for a CT test, set in ${CT_CFG},
-# sets ${TREE_IDS} and provisions logs for them.
+# Builds configuration for a CT test, set in ${CT_CFG} and provisions logs for
+# them.
 set -e
+
+if [[ $# -ne 1 ]]; then
+  echo "Usage: $0 <admin_endpoint>"
+  exit 1
+fi
+ADMIN_ENDPOINT="$1"
+
 INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
 . "${INTEGRATION_DIR}"/common.sh
 
 # Build config file with absolute paths
 CT_CFG=$(mktemp "${INTEGRATION_DIR}"/ct-XXXXXX)
-sed "s!@TESTDATA@!${TESTDATA}!" ./integration/ct_integration_test.cfg > ${CT_CFG}
-
-# Retrieve tree IDs from config file
-TREE_IDS=$(grep LogID ${CT_CFG} | grep -o '[0-9]\+')
-for id in ${TREE_IDS}; do
-    echo "Provisioning test log (Tree ID: ${id}) in database"
-    "${SCRIPTS_DIR}"/wipelog.sh ${id}
-    "${SCRIPTS_DIR}"/createlog.sh ${id}
+sed "s!@TESTDATA@!${TESTDATA}!" ./integration/ct_integration_test.cfg > "${CT_CFG}"
+
+echo 'Building createtree'
+go build ${GOFLAGS} ./cmd/createtree/
+
+num_logs=$(grep -c '@TREE_ID@' "${CT_CFG}")
+for i in $(seq ${num_logs}); do
+  pem_file=$(grep -m$i 'PrivKeyPEMFile' "${CT_CFG}" | tail -1 | awk -F'"' '{print $4}')
+  pem_pass=$(grep -m$i 'PrivKeyPassword' "${CT_CFG}" | tail -1 | awk -F'"' '{print $4}')
+  echo "Creating tree with PEM key: ${pem_file}"
```

Comment:
This is using the wrong key, I think.  The CT personality has its own per-log keys, which are different than the ones used by the Trillian Log server; what's in the CT log config file is the former, not the latter.

So for now this could just use `testdata/log-rpc-server.privkey.pem` for provisioning throughout, with a TODO of some sort.

(The CT personality config will need to be extended to hold the Log's *public* key in future, but should never get its private key.)